app/config/errors.php

```diff
@@ -728,4 +728,9 @@
         'description' => 'Migration is already in progress. You can check the status of the migration in your Appwrite Console\'s "Settings" > "Migrations".',
         'code' => 409,
     ],
+    Exception::MIGRATION_PROVIDER_ERROR => [
+        'name' => Exception::MIGRATION_PROVIDER_ERROR,
+        'description' => 'An error occurred on the provider\'s side. Please try again later.',
+        'code' => 424,
```

Comment:
Let's use a 400 error here...

Reply:
Can you also make sure the frontend is handling and displaying it correctly ?

Reply:
Frontend isn't currently displaying this correctly, bringing it up to the frontend team